packages/docs/src/vite-plugin-synergy-styles/index.ts

```diff
@@ -66,6 +71,7 @@ export default function vitePluginSynergyStyles(
         return undefined;
       }
 
+      runStylesBuild();
```

Comment:
I am missing this on the other outputs where we are recreating the manifest, e.g. `configureServer`. I get that on `generateBundle` this should be already done, but I am not sure about configureServer.

Reply:
I just forgot the others, because for the `pnpm start` I only needed it in the `load` function 😅